service/kas/access/rewrap.go

```diff
@@ -766,6 +797,7 @@ func (p *Provider) tdf3Rewrap(ctx context.Context, requests []*kaspb.UnsignedRew
 			}
 
 			if !access {
+				// ? Maybe we should change the error if we get an obligation failure? Not just "forbidden"
```

Comment:
definitely a more precise AUDIT event.  the client response IMO is fine

Reply:
Audit will be populated with the obligations involved in a decision from the auth service side because every decision result gets logged to audit alongside the rewrap audit logs (important to track key releases).